lib/jekyll-remote-theme/munger.rb

```diff
@@ -22,6 +22,7 @@ def munge!
         Jekyll.logger.info LOG_KEY, "Using theme #{theme.name_with_owner}"
         return theme if munged?
 
+        Jekyll.logger.info LOG_KEY, "Downloading..."
```

Comment:
Was going to suggest this live in the Downloader so it's always called when there's an actual download, but then saw https://github.com/benbalter/jekyll-remote-theme/blob/master/lib/jekyll-remote-theme/downloader.rb#L42. Would it make sense to raise that log entry to the level of info, or potentially have two, one with the URL/Path, one without?

Reply:
Hey @benbalter , the change you suggested sounds good (better) also. I'm not as across the Ruby ecosystem so you would be better placed to suggest a suitable log message.

The reason I proposed this PR was that when I saw my Jekyll theme was saying "Using Remote Theme" and didn't launch the local web server, I thought something was wrong, only after digging in the source did I realise I should be more patient for it to download something :)

Reply:
If you move this line to the line lined above, I'd be glad to merge this pull request! :smile: